/home/shambhuraj/sam/github/SfM-and-NeRF/sfm_n_nerf/Phase2/Wrapper.py:58: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  ray_origin = torch.tensor(ray_origin_in_world_coordinate)
Traceback (most recent call last):
  File "/home/shambhuraj/sam/github/SfM-and-NeRF/sfm_n_nerf/Phase2/Wrapper.py", line 296, in <module>
    main(args)
  File "/home/shambhuraj/sam/github/SfM-and-NeRF/sfm_n_nerf/Phase2/Wrapper.py", line 260, in main
    train(images, poses, camera_info, args)
  File "/home/shambhuraj/sam/github/SfM-and-NeRF/sfm_n_nerf/Phase2/Wrapper.py", line 229, in train
    torch.save({
  File "/home/shambhuraj/.local/lib/python3.10/site-packages/torch/serialization.py", line 618, in save
    with _open_zipfile_writer(f) as opened_zipfile:
  File "/home/shambhuraj/.local/lib/python3.10/site-packages/torch/serialization.py", line 492, in _open_zipfile_writer
    return container(name_or_buffer)
  File "/home/shambhuraj/.local/lib/python3.10/site-packages/torch/serialization.py", line 463, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
RuntimeError: [enforce fail at inline_container.cc:462] . invalid file name: ./Phase2/checkpoints/
Traceback (most recent call last):
  File "/home/shambhuraj/sam/github/SfM-and-NeRF/sfm_n_nerf/Phase2/Wrapper.py", line 296, in <module>
    main(args)
  File "/home/shambhuraj/sam/github/SfM-and-NeRF/sfm_n_nerf/Phase2/Wrapper.py", line 260, in main
    train(images, poses, camera_info, args)
  File "/home/shambhuraj/sam/github/SfM-and-NeRF/sfm_n_nerf/Phase2/Wrapper.py", line 229, in train
    torch.save({
  File "/home/shambhuraj/.local/lib/python3.10/site-packages/torch/serialization.py", line 618, in save
    with _open_zipfile_writer(f) as opened_zipfile:
  File "/home/shambhuraj/.local/lib/python3.10/site-packages/torch/serialization.py", line 492, in _open_zipfile_writer
    return container(name_or_buffer)
  File "/home/shambhuraj/.local/lib/python3.10/site-packages/torch/serialization.py", line 463, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
RuntimeError: [enforce fail at inline_container.cc:462] . invalid file name: ./Phase2/checkpoints/